### ISSUE TITLE
il8n support missing from register.inc.php

### ISSUE THREAD

**Issue by stevland** [osTicket/osTicket#4965]
\include\client\register.inc.php

```
<input type="submit" value="Register"/>
<input type="button" value="Cancel" onclick="javascript:
        window.location.href='index.php';"/>
```

should be:

```
<input type="submit" value="<?php echo __('Register'); ?>"/>
<input type="button" value="<?php echo __('Cancel'); ?>" onclick="javascript:
        window.location.href='index.php';"/>
```

**Comment by JediKev** [comment]
@stevland 

Thank you for the suggestion. We are going to push all the new strings to crowdin soon so more suggestions are welcome. 👍🏻

Cheers.

**Comment by JediKev** [comment]
@stevland 

This is addressed in #5024, thanks again.

Cheers.
